### PR TITLE
Update SentimentAnalysis tutorial with new datasets

### PR DESCRIPTION
**Pull request by JRAlexander**
# Update SentimentAnalysis tutorial with new datasets

Updated the tutorial with the Wikidetox data sets.

[Internal Review URL](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/sentiment-analysis?branch=pr-en-us-5616)

### CODE REVIEW

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 description: Discover how to use ML.NET in a classification scenario to understand how to use sentiment prediction to take the appropriaste action.
-ms.date: 05/21/2018
+ms.date: 05/24/2018
 ms.custom: mvc
 #Customer intent: As a developer, I want to use ML.NET to apply a binary classification task so that I can understand how to use sentiment prediction to take appropriaste action.
 ---
 # Tutorial: Use the ML.NET APIs in a sentiment analysis classification scenario
 
+> [!NOTE]
+> This topic refers to ML.NET, which is currently in Preview, and material may be subject to change. For more information, please visit [the ML.NET introduction](https://www.microsoft.com/net/learn/apps/machine-learning-and-ai/ml-dotnet).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: remove please. https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/p/please

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 2]
```diff
@@ -15,16 +18,12 @@ In this tutorial, you learn how to:
...
 > * Evaluate the model with a different dataset
 
 ## Sentiment analysis sample overview
 
-The sample is a console app that uses the ML.NET API to train a model that classifies and predicts sentiment as either positive or negative. It also evaluates the model with a second dataset for quality analysis. The sentiment datasets are from University of California, Irvine (UCI).
-
-Prediction and evaluation results are displayed accordingly so that analysis and action can be taken.
-
-Sentiment analysis is either positive or negative. So, you can use classification to train the model, for prediction, and for evaluation.
+The sample is a console app that uses the ML.NET API to train a model that classifies and predicts sentiment as either positive or negative. It also evaluates the model with a second dataset for quality analysis. The sentiment datasets are from  the WikiDetox.
```
Comment (mairaw) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: ML.NET APIs?

remove extra space between from and the 

link to WikiDetox?

File: docs/machine-learning/tutorials/sentiment-analysis.md [hunk 3]
```diff
@@ -52,13 +51,17 @@ You then need to **determine** the sentiment, which helps you with the machine l
...
+
+* Please refrain from adding nonsense to Wikipedia.
+* He is the best, and the article should say that.
 
 ## Prerequisites
 
 * [Visual Studio 2017 15.6 or later](https://www.visualstudio.com/downloads/?utm_medium=microsoft&utm_source=docs.microsoft.com&utm_campaign=button+cta&utm_content=download+vs2017) with the ".NET Core cross-platform development" workload installed.
 
-* [The UCI Sentiment Labeled Sentences dataset zip file](https://archive.ics.uci.edu/ml/machine-learning-databases/00331/sentiment%20labelled%20sentences.zip)
+* The [Wikipedia detox line data tab seperated file (wikiPedia-detox-250-line-data.tsv)](https://github.com/dotnet/machinelearning/blob/master/test/data/wikipedia-detox-250-line-data.tsv).
```
Comment (mairaw) [inline docs/machine-learning/tutorials/sentiment-analysis.md]: typo: seperated

**Review by mairaw — APPROVED** [review]
LGTM modulo the small fixes I found

**Comment by JRAlexander** [comment]
closing and reopening to restart hung build

**Pull request merged**